tutorials/tutorial-partIII.md

```diff
@@ -68,18 +68,19 @@ two()
 
 #### height(h) and solid(true/false)
 
-specify the default value for Constructive primitive tube() and box(). (solid() only affects tube(), tubFast(), and  tubeShell() not the box()) ) So inside of the g() block above, the h and solid arguments can be ommited when using box() or tube()  
+Specify the default value for Constructive primitive tube() and box(). (solid() only affects tube(), tubFast(), and  tubeShell() not the box()). So inside of the g() block above, the h and solid arguments can be omitted when using box() or tube().
 
 ------------------------------------------------
 
 ## assembling mechanical Parts from several Modules
 
 #### assemble()
 
-allows application of universal operations like adding and removing solids (i.e. boring holes or addings screws), only to a Part of the Model representing a specific mechanical (sub-)Part)
+Allows application of universal operations like adding and removing solids (i.e. bring holes or adding screws), only to a Part of the Model representing a specific mechanical (sub-Part).
```

Comment:
boring as in "bore  a hole" or "make a  bore" or a "boring company" latter (c) Elon Musk :)

Reply:
oops :)